### PR TITLE
applications: nrf5340_audio: Improvements from UPF London

### PR DESCRIPTION
**Pull request by alexsven**
Several commits here

### CODE REVIEW

File: applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c [hunk 1]
```diff
@@ -119,6 +119,74 @@ static struct bt_bap_lc3_preset lc3_preset_source_16_2_1 =
 
 static bool playing_state = true;
 
+static void print_codec(const struct bt_audio_codec_cfg *codec, enum bt_audio_dir dir)
```
[the remark this answers is not in the document]
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_stream/unicast/unicast_client.c:122]: Yes, I can move it to le_audio.c

**Review by koffes — APPROVED** [review]

File: applications/nrf5340_audio/child_image/hci_ipc.conf [hunk 2]
```diff
@@ -30,6 +30,10 @@ CONFIG_BT_CTLR_ADV_DATA_LEN_MAX=251
 CONFIG_MPSL_TRIGGER_IPC_TASK_ON_RTC_START=y
 CONFIG_MPSL_TRIGGER_IPC_TASK_ON_RTC_START_CHANNEL=4
 
+# Needed for builds with nrf21540
+#CONFIG_BT_CTLR_TX_PWR_ANTENNA=20
```
Comment (rick1082) [inline applications/nrf5340_audio/child_image/hci_ipc.conf:34]: Maybe we can set these to 10 dBm

File: applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c [hunk 3]
```diff
@@ -36,6 +36,16 @@ static size_t per_adv_local_size;
 /* Bonded address queue */
 K_MSGQ_DEFINE(bonds_queue, sizeof(bt_addr_le_t), BONDS_QUEUE_SIZE, 4);
 
+static struct bt_le_adv_param adv_param = {
```
Comment (rick1082) [inline applications/nrf5340_audio/src/bluetooth/bt_management/advertising/bt_mgmt_adv.c:39]: Would be great if we can add some word for identify this is for ADV-EXT.

File: applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c [hunk 4]
```diff
@@ -140,3 +140,71 @@ int le_audio_stream_dir_get(struct bt_bap_stream const *const stream)
 
 	return ep_info.dir;
 }
+
+void le_audio_print_codec(const struct bt_audio_codec_cfg *codec, enum bt_audio_dir dir)
```
Comment (rick1082) [inline applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c:144]: Not very necessary, we can handle this in another PR: Could this API also be reused in broadcast sink?

**Review by rick1082 — APPROVED** [review]
Great PR with many fixes!

File: applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c (hunk 4 above)
Comment (alexsven) [reply applications/nrf5340_audio/src/bluetooth/bt_stream/le_audio.c:144]: I did look into it, but it requires quite a bit of rewrite, so we should do it in a separate PR

File: applications/nrf5340_audio/child_image/hci_ipc.conf (hunk 2 above)
Comment (alexsven) [reply applications/nrf5340_audio/child_image/hci_ipc.conf:34]: It's only comments, but I can change to 10 and then add a comment about 20

**Pull request merged**